runners/dolphin/build.sh

```diff
@@ -22,7 +22,7 @@ arch=$(uname -m)
 repo_url="https://github.com/dolphin-emu/dolphin"
 
 InstallBuildDependencies() {
-    install_deps cmake libxext-dev libreadline-dev libgl1-mesa-dev libevdev-dev \
+    sudo apt install -y cmake libxext-dev libreadline-dev libgl1-mesa-dev libevdev-dev \
```

Comment:
For installing the dependencies, you'll have to run the script itself as root.

Reply:
Not really. We have instruction in README that allow you to not do that.

Reply:
Also, this is how Wine build script currently installs then.